src/components/ConfessButton/ConfessButton.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import Button from '../Button';
+import { SaltGrinder } from '../Icons';
+import { colorRed } from '../../styles/designTokens';
+
+const RedButton = styled(Button)`
+  background: ${colorRed};
+`;
+
+function ConfessButton() {
+  return (
+    <RedButton label="Confess my mistake">
+      <SaltGrinder />
```

Comment:
In same directory I wold create wrapper Icon component (div) which positions absolutely SaltGrinder. I would rather use some a bit more generic names, so it could be StyledButton rather than Red one.
Pass it as icon prop to previously created Button:
```
<StyledButton icon={<Icon />}>Confess my mistake</StyledButton>
```